tests/test_utilities.py

```diff
@@ -57,7 +57,7 @@ def get_sample(sample_file: str, function_group: str) -> Union[str, bytes]:
     result_samples = f"{function_group}/{function_group}_result_samples"
     request_samples = f"{function_group}/{function_group}_request_samples"
     log_samples = f"{function_group}/{function_group}_log_samples"
-    if f"{function_group}_result" in sample_file:
+    if "_result" in sample_file:
```

Comment:
Is this change necessary now that we are not sharing test data between function groups?

Reply:
technically not. Names of "common test files could be changed to all have "common_result" in their name when necessary, but I think this makes it harder to distinguish what sort of admin activity is occurring in the common result files, so i thought that keeping the "user_result" or "resource_result" in the names worked best, but with that in place, without the above change, the code could not find them.

Reply:
Ok, I see, the problem is that the sample files in common don't start with `common`. In that case, I think this change makes sense.